cmd/diag/ui/ui.go

```diff
@@ -0,0 +1,157 @@
+package ui
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os/exec"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/ledgerwatch/erigonwatch"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/go-chi/cors"
+	"github.com/ledgerwatch/erigon/cmd/diag/flags"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	UIPortFlag = cli.IntFlag{
+		Name:     "ui.port",
+		Aliases:  []string{"up"},
+		Usage:    "UI port to listen on",
+		Required: false,
+		Value:    8080,
+	}
+)
+
+var Command = cli.Command{
+	Name:      "ui",
+	Action:    runUI,
+	Aliases:   []string{"u"},
+	Usage:     "run local ui",
+	ArgsUsage: "",
+	Flags: []cli.Flag{
+		&flags.DebugURLFlag,
+		&UIPortFlag,
+	},
+	Description: ``,
+}
+
+func runUI(cli *cli.Context) error {
+	supportedSubpaths := []string{
+		"sentry-network",
+		"sentinel-network",
+		"downloader",
+		"logs",
+		"chain",
+		"data",
+		"debug",
+		"testing",
+		"performance",
+		"documentation",
+		"issues",
+		"admin",
+	}
+
+	listenAddr := "127.0.0.1"
```

Comment:
Should be configurable via flag ideally.

Reply:
The main idea to have local UI is to simplify connecting to node, so main idea is to serve UI on localhost. If you have any examples of use cases which require to serve on different address from localhost please share with me.

Reply:
There's a case where you have Erigon running on private VMs and you don't want to expose the ports to public. So you can have this UI running on a machine that has access to this private network and is only accessible to the public via port 80/443.

Reply:
done